src/cplscheme/SerialCouplingScheme.cpp

```diff
@@ -122,10 +134,8 @@ bool SerialCouplingScheme::exchangeDataAndAccelerate()
     sendData(getM2N(), getSendData());
     // the second participant does not want new data in the last iteration of the last time window
     if (isCouplingOngoing() || (isImplicitCouplingScheme() && not convergence)) {
-      if (_participantReceivesTimeWindowSize) {
-        receiveAndSetTimeWindowSize();
-      }
       PRECICE_DEBUG("Receiving data...");
+      receiveAndSetTimeWindowSize();
```

Comment:
What happened to `_participantReceivesTimeWindowSize`?
Also, the location of the debug statement seems wrong now.

Reply:
I moved it inside of `receiveAndSetTimeWindowSize()` to make it consistent with `sendTimeWindowSize()`. Moving the statement inside of the function saves some checks in the functions using `receiveAndSetTimeWindowSize`.

I'll move the debug statement.